kubernetes/helm/presto/values.yaml

```diff
@@ -32,6 +32,7 @@ pinot:
   useDateTrunc: true
   inferDateTypeInSchema: true
   inferTimestampTypeInSchema: true
+  useStreamingForSegmentQueries: false
```

Comment:
in values.yaml, you can directly set config files:
```
catalogs:
    pinot.properties: |-
        connector.name=pinot
        pinot.controller-urls=pinot-controller:9000
        pinot.controller-rest-service=pinot-controller:9000
        pinot.allow-multiple-aggregations=true
        pinot.use-date-trunc=true
        pinot.infer-date-type-in-schema=true
        pinot.infer-timestamp-type-in-schema=true
        pinot.use-streaming-for-segment-queries=true
```

Reply:
Done